src/main/resources/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.durabletask.executors.OnceRetentionStrategy
+
+def f = namespace(lib.FormTagLib);
+
+f.entry(title: "Idle timeout", field: "idleMinutes") {
+    f.number(default: 0)
```

Comment:
It caused many issues in docker plugin, as soon as strategy runs it kills slave. I reverted to 10 minutes according to users feedback.

FYI @stephenc